requirements-minimal.txt

```diff
@@ -14,3 +14,4 @@ neo>=0.9.0
 roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@c23e4a00324b381f2aad25d685e1cd68b588b563  # v0.4.7, pre-transpose fix
 lxml>=4.6.5
 scipy>=1.4.1
+click  # must be unspecified version to handle platform/pyVersion
```

Comment:
I was thinking that loose dependency should work but you tried that, how comes?

Reply:
This is something you have to do when the package has different min/max version builds available on different platforms , so this allows it to infer which one is proper for a given setup

We also use it for sonpy since it has similar problems